src/devices/raex.ts

```diff
@@ -0,0 +1,15 @@
+import {battery, identify, windowCovering, commandsOnOff} from '../lib/modernExtend';
+import {Definition} from '../lib/types';
+
+const definitions: Definition[] = [
+    {
+        zigbeeModel: ['TQL25-2211'],
+        model: 'TQL25-2211',
+        vendor: 'Raex',
+        description: 'Tubular motor',
+        extend: [battery(), identify(), windowCovering({controls: ['lift']}), commandsOnOff({commands: ['on', 'off', 'toggle']})],
```

Comment:
Does this device provide any `action` values? Otherwise `commandsOnOff` can be removed.

Reply:
Good call, no `action` values are supported. Also removed `identify` as motor did not respond to that command.